northstar/src/api/client.rs

```diff
@@ -446,13 +446,34 @@ impl<'a, T: AsyncRead + AsyncWrite + Unpin> Client<T> {
     /// # async fn main() {
     /// #   let mut client = Client::new(tokio::net::TcpStream::connect("localhost:4200").await.unwrap(), None, Duration::from_secs(10)).await.unwrap();
     /// let npk = Path::new("test.npk");
-    /// client.install(&npk, "default").await.expect("failed to install \"test.npk\" into repository \"default\"");
+    /// client.install_file(npk, "default").await.expect("failed to install \"test.npk\" into repository \"default\"");
     /// # }
     /// ```
-    pub async fn install(&mut self, npk: &Path, repository: &str) -> Result<Container, Error> {
-        self.fused()?;
+    pub async fn install_file(&mut self, npk: &Path, repository: &str) -> Result<Container, Error> {
         let file = fs::File::open(npk).await.map_err(Error::Io)?;
         let size = file.metadata().await?.len();
```

Comment:
Looks like there's a `#[from] io::Error` somewhere in `client::Error`. Means the `map_err(Error::Io)` should not be needed.